[2022-06-16 10:32:31,605] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: analytics.getdata scheduled__2022-05-24T00:00:00+00:00 [queued]>
[2022-06-16 10:32:31,660] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: analytics.getdata scheduled__2022-05-24T00:00:00+00:00 [queued]>
[2022-06-16 10:32:31,663] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-16 10:32:31,664] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-06-16 10:32:31,668] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-16 10:32:31,754] {taskinstance.py:1377} INFO - Executing <Task(MySqlOperator): getdata> on 2022-05-24 00:00:00+00:00
[2022-06-16 10:32:31,785] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'analytics', 'getdata', 'scheduled__2022-05-24T00:00:00+00:00', '--job-id', '444', '--raw', '--subdir', 'DAGS_FOLDER/testing_etl.py', '--cfg-path', '/tmp/tmp35ecthr1', '--error-file', '/tmp/tmpoa3uw5x3']
[2022-06-16 10:32:31,786] {standard_task_runner.py:80} INFO - Job 444: Subtask getdata
[2022-06-16 10:32:31,767] {standard_task_runner.py:52} INFO - Started process 3585 to run task
[2022-06-16 10:32:32,163] {task_command.py:370} INFO - Running <TaskInstance: analytics.getdata scheduled__2022-05-24T00:00:00+00:00 [running]> on host fe209149cfc1
[2022-06-16 10:32:33,015] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=analytics
AIRFLOW_CTX_TASK_ID=getdata
AIRFLOW_CTX_EXECUTION_DATE=2022-05-24T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-24T00:00:00+00:00
[2022-06-16 10:32:33,017] {mysql.py:82} INFO - Executing: SELECT * from Customer_T;
[2022-06-16 10:32:33,148] {base.py:68} INFO - Using connection ID 'mysql_con' for task execution.
[2022-06-16 10:32:33,261] {dbapi.py:213} INFO - Running statement: SELECT * from Customer_T;, parameters: None
[2022-06-16 10:32:33,264] {dbapi.py:221} INFO - Rows affected: 19
[2022-06-16 10:32:33,386] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=analytics, task_id=getdata, execution_date=20220524T000000, start_date=20220616T103231, end_date=20220616T103233
[2022-06-16 10:32:33,570] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-16 10:32:34,245] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
